notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Optimization is done then on MC gammas vs. OFF, right? - this should be clarified at the beginning</p><p>It may not be the one resulting in best sensitivity for real data, if MC and data do not agree well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Yes, optimization is done in MC events. Now I included MC protons to calculate MC sensitivity vs real data sensitivity, and therefore cuts are optimized using MC gammas and protons. </p><p>I'm not sure how to optimize the cuts based on real data only. The way the cut optimization function in pyirf works takes the events for the "signal" and "background". With MC that's easy because gammas are signal, protons are background. </p><p>With real data however signal=on-alpha*off, and I don't have a way to separate which events are signal and which are background... Also, the cut optimizacion is based on gamma efficiency, and again I don't know what is the gamma efficiency after cuts in real data... maybe I'm missing something, but I'm comfused. </p>